extension/src/ui/components/VideoDataSyncDialog.tsx

```diff
@@ -186,6 +204,8 @@ export default function VideoDataSyncDialog({
                                     return newSelectedSubtitles;
                                 })
                             }
+                            onFocus={() => setIsDropdownActive(true)}
+                            onBlur={() => setIsDropdownActive(false)}
```

Comment:
As you pointed out, with this implementation it still takes another click for the dialog to respond to the `Enter` key. I think for this change to be really useful the dialog should close on `Enter` as long as none of the drop-downs are open. This would reduce the total clicks required to select subtitles.